share/spice/sports/mlb/games/sports_mlb_games.js

```diff
@@ -0,0 +1,162 @@
+(function (env) {
+    "use strict";
+
+    env.ddg_spice_sports_mlb_games = function(apiResult) {
+
+        if (!apiResult || !apiResult.data || !apiResult.data.games || !apiResult.data.games.length) {
+            return Spice.failed('mlb_games');
+        }
+        
+        DDG.require('moment.js', function(){
+            Spice.add({
+                id: 'mlb_games',
+                name: 'MLB Games',
+
+                allowMultipleCalls: true,
```

Comment:
Is this right? The api doesn't return a next param in the response so there's no load more tile currently showing.